webapp/components/wpt-results.html

```diff
@@ -401,16 +406,30 @@
             displayedNodeMap.set(dirPath, {isDir: isDir, results: {}});
           }
           let results = this.testFiles[testFileName];
-          Object.keys(results).forEach(resultURL => {
-            if (!(resultURL in displayedNodeMap.get(dirPath).results)) {
-              displayedNodeMap.get(dirPath).results[resultURL] = {passing: 0, total: 0};
-            }
-            displayedNodeMap.get(dirPath).results[resultURL].passing += results[resultURL][0];
-            displayedNodeMap.get(dirPath).results[resultURL].total += results[resultURL][1];
-          });
+          if (results) {
+            Object.keys(results).forEach(resultURL => {
+              if (!(resultURL in displayedNodeMap.get(dirPath).results)) {
+                displayedNodeMap.get(dirPath).results[resultURL] = {passing: 0, total: 0};
+              }
+              displayedNodeMap.get(dirPath).results[resultURL].passing += results[resultURL][0];
+              displayedNodeMap.get(dirPath).results[resultURL].total += results[resultURL][1];
+            });
+          }
         };
 
-        Object.keys(this.testFiles).forEach(testFileName => {
+        let paths = Object.keys(this.testFiles);
+        if (this.manifest) {
+          paths = [];
+          for (const files of TEST_TYPES.map(i => this.manifest.items[i]).filter(i => i)) {
+            for (const items of Object.values(files)) {
+              for (const item of items) {
+                paths.push(item[0]); // First item array value is url
```

Comment:
Yes, this is the good stuff! https://manifest-test-list-dot-wptdashboard-staging.appspot.com/results/ now includes accname and other directories with only manual tests, and all my manual tests in https://manifest-test-list-dot-wptdashboard-staging.appspot.com/results/fullscreen/api.

Visually, the missing results are just white boxes, and I think it becomes important with this change to show the not-run tests differently somehow. Maybe just count directories with no tests as 0 / 0? At the file level, I guess "not run"?

The subtest level is a pre-existing problem. NOTRUN is a possible status for tests found but not run, so maybe call it "missing", like the stability checker does? I suppose that "missing" at the other levels would also work.

WDYT?

Reply:
Happy to tweak appearance. Could you whip up some mocks?